daffodil-lib/src/main/scala/org/apache/daffodil/exceptions/ThrowsSDE.scala

```diff
@@ -82,32 +83,29 @@ trait ThrowsSDE {
 trait SavesErrorsAndWarnings {
 
   def SDE(id: String, args: Any*): Nothing
```

Comment:
We should probably also have a similar ErrorID for SDE's. That would be the first step towards adding support for something like JALoP or various other auditing mechanisms. Not worth doing as part of this since there's no use for those ID's like there are for warn IDs, but something to think about for the future. Could be a good task for a new developer.

Reply:
See also JIRA DAFFODIL-628 (JAF Logging).